### ISSUE TITLE
allow using "get" on all updates shown by "list"

### ISSUE THREAD

**Issue by mvdan** [icholy/gomajor#19]
If `gomajor list` shows me a few updates, either minor or major version bumps, it would be nice to be able to use `gomajor get` with those directly. Perhaps there's a way, but I haven't found one via the help docs, so I manually copy-paste the new module paths into `gomajor get`.

Perhaps `gomajor get all`?

**Comment by icholy** [comment]
That sounds reasonable. You can do something like this until that's implemented 
``` sh
$ gomajor list | cut -d: -f1 | xargs -I{} gomajor get {}@latest
```

**Comment by mvdan** [comment]
One more idea: it would be nice to be able to "get all major updates", i.e. `gomajor get` the result of `gomajor list -major`.

**Comment by icholy** [comment]
I think it makes sense for `gomajor get all` to support all the same flags as `gomajor list`.

**Comment by icholy** [comment]
This is implemented in `v0.8.0`.
